app/webroot/js/misp.js

```diff
@@ -3276,93 +3276,143 @@ function getRemoteSyncUser(id) {
 function testConnection(id) {
     $.ajax({
         url: '/servers/testConnection/' + id,
-        type:'GET',
-        beforeSend: function (XMLHttpRequest) {
-            $("#connection_test_" + id).html('Running test...');
+        type: 'GET',
+        beforeSend: function () {
+            $("#connection_test_" + id).text('Running test...');
         },
         error: function(){
-            $("#connection_test_" + id).html('Internal error.');
-        },
-        success: function(response){
-            var result = response;
+            $("#connection_test_" + id).html('<span class="red bold">Internal error</span>');
+        },
+        success: function(result) {
+            function line(name, value, valid) {
```

Comment:
Could you use a safer way to generate the line like using jQuery's `.text()` function?
Because if one manages to craft a certificate with HTML inside it (which is easy), he/she could do an XSS.